etc/config/assembly.amazon.properties

```diff
@@ -66,6 +66,71 @@ compiler.gnuassnapshot.exe=/opt/compiler-explorer/gcc-snapshot/bin/as
 compiler.gnuassnapshot.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
 compiler.gnuassnapshot.semver=(trunk)
 
+
+group.gnuasarm.compilers=gnuasarmhfg54:gnuasarmg454:gnuasarmg464:gnuasarmg630:gnuasarmg640:gnuasarmg730:gnuasarmg820:gnuasarm930:gnuasarm1020
+group.gnuasarm.versionFlag=--version
+group.gnuasarm.options=-g
+group.gnuasarm.isSemVer=true
+group.gnuasarm.baseName=ARM gcc
```

Comment:
Should we perhaps name this `gas` instead of `gcc`? It's not part of gcc, it comes from the binutils. though I see we have an entry for each GCC...hmm, probably should be tagged with the `gas` version? Not hugely wed to this though as it's at least defensibly "the version of GAS that we use with version x.y.z of GCC" - though that's a choice we made, not something the compiler necessarily mandates.

Reply:
It's technically true, but I think it might be wise to keep GCC for comparison value.
Just so you can copy/paste anything that comes out of Gcc x and compile it with the assembly compiler on the same version.

But ofc you can always check the actual AS version via the version information icon.

Reply:
Sounds good to me! Thanks!